### PR TITLE
Implement Jenkins pipeline for CI/CD

### PR DESCRIPTION
**Pull request by Log234**
Implements a Jenkins pipeline which builds on new commits in GitHub and publishes to api.lavinia.no when the commits are on the master branch.

### CODE REVIEW

**Review by simendag — CHANGES_REQUESTED** [review]
I don't see tests being run here. Tests in addition need to not give a "failed build"-result in Jenkins but a yellow build. A failed build does not produce any artefacts. A failed test may/may not publish artefacts, but they should not be deployed.

File: Jenkinsfile
Comment (simendag) [inline Jenkinsfile:39]: What's this madness? This is not the kind of thing that stands up to a security review.

Comment (simendag) [reply Jenkinsfile:39]: When you use SSH and don't check the host key, you have in principle no idea what computer you just connected to.

**Comment by simendag** [comment]
It would be strongly preferred if we fixed the not-checking-hostkey thing before we merge it. Being that this is an open source project it's a pretty glaring security problem if someone does get into the network one way or another.

**Comment by simendag** [comment]
This is ancient - should we just close it? I imagine you got this running in the end. :-P Everything I complained about was addressed anyway.

**Comment by Log234** [comment]
> 
> 
> This is ancient - should we just close it? I imagine you got this running in the end. :-P Everything I complained about was addressed anyway.

Agreed, let's close this PR.

**Comment by Log234** [comment]
Why did we actually close this? We clearly do not have any Jenkins file in the development or master branch...?

**Review by simendag — APPROVED** [review]
Seems fine - will need an update for upgrade to .NET 6 on later occasion though, since we are not defining our own output for the release build 👍

**Pull request merged**
